YIF.Core.Service/Concrete/Services/ValidatorServices/ValidationService.cs

```diff
@@ -40,6 +40,7 @@ public LoginValidator(UserManager<DbUser> userManager, IRecaptchaService recaptc
 
             RuleFor(x => x.Email).Must(IsEmailExist).WithMessage("Логін або пароль неправильний!");
             RuleFor(x => x.Password).Must(IsPasswordCorrect).WithMessage("Логін або пароль неправильний!");
+            RuleFor(x => x.Email).Must(IsUserNotDeleted).WithMessage("Користувач був видалений!");
```

Comment:
Maybe we should put this rule on top of validation, because we have cascade mode, so it will be good to check if user deleting status in the first place.